### ISSUE TITLE
Compilation issues with BoxFile.cls

### ISSUE THREAD

**Issue by mahavrilla** [box/box-salesforce-sdk#40]
When trying to deploy via Deploy to Salesforce Functionality, the deploy fails with multiple errors all pointing to BoxFile.cls

`classes/BoxFile.cls(263,14):Missing ';' at 'responseBody'`

There is a missing assignment at line 263. In 
`String responseBody BoxApiRequest.ensureBoxResourceResponseBody(response, 'BoxFile.getTrashedFile', new set<Integer>{200});`

Should be 

`String responseBody =  BoxApiRequest.ensureBoxResourceResponseBody(response, 'BoxFile.getTrashedFile', new set<Integer>{200});
`
If this issue is not already being addressed, I'll create a PR.

**Comment by sukgupta** [comment]
Also, the method - ensureBoxResourceResponseBody does not exist, so even after this fix, the deployment is still failing for me.

**Comment by jasonpan12** [comment]
Hey @sukgupta , good catch. I actually ran into the same issue too, but unfortunately it is noted in the documentation that the Deploy to Salesforce functionality is not owned or operated by Box. I instead was able to get it installed via the direct Package Links. I also made a PR to remove the reference to the Deploy to Salesforce functionality if it doesn't work: https://github.com/box/box-salesforce-sdk/pull/41
